### DISCUSSION TITLE
Negative Power Day/total Counter

### DISCUSSION THREAD

**Discussion by Massimo65** [arendst/Tasmota#13359] [Q&A] [answered]
Good morning
I have installed the latest version 9.5.0.9 from 12-10-2021. when the power values ​​go negative, the day and total values ​​are no longer correct. any suggestion? Thank you
![Schermata del 2021-10-14 10-50-09](https://user-images.githubusercontent.com/603217/137284096-f291f31f-28ee-4090-a652-1fff0ac599b4.png)

**Comment by arendst** [comment, marked as answer]
See relevant issue with possible solution.

**Comment by ascillato** [comment]
@Massimo65

Hi, were you able to test the fix? https://github.com/arendst/Tasmota/issues/13363 It is already merged into Tasmota codebase. Thanks.
